### ISSUE TITLE
TikolaNesla from SuperUser. Thanks for the code!

### ISSUE THREAD

**Issue by kart-archive** [FuPeiJiang/mute-virtual-desktop#1]
Hey!
Found this really helpful. Just wanted to thank you. Don't see a lota nice guys like you often on the internet 

Fellow Netizen,
Karthik

**Comment by FuPeiJiang** [comment]
np

I've made a fix to not confuse apps, you should update

**Comment by FuPeiJiang** [comment]
well you see, I didn't make nircmd.exe, and I've used code from 2 ahk forum posts to make VD.ahk, so you've been indirectly helped by MANY people.

**Comment by kart-archive** [comment]
That's true but tbh it would have taken me at least 15 minutes to figure out the syntax and it was nice of you to help. Hence the bounty 😉
